easybuild/easyconfigs/p/pyarrow-hotfix/pyarrow-hotfix-0.6-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonBundle'
+
+name = 'pyarrow-hotfix'
+version = '0.6'
+
+homepage = 'https://senexcanis.com/open-source/cppheaderparser'
+description = """CppHeaderParser is a pure python module that will parse C++
+header files and generate a data structure representing the class."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+dependencies = [
+    ('binutils', '2.38'),
+    ('Python', '3.10.4'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+_preinstallopts = """sed -i -e 's/^build-backend = .*/build-backend = "setuptools.build_meta"/g' """
+_preinstallopts += """-e 's/^requires = .*/requires = ["setuptools"]/g' """
+_preinstallopts += """-e 's/dynamic = \["version"\]/version = "%(version)s"/g' pyproject.toml && """  # noqa
```

Comment:
I'm unsure about what the recommended best practices are, but my opinion is that this would be better as a patch file.

https://docs.easybuild.io/writing-easyconfig-files/#common_easyconfig_param_sources_patches

Reply:
I see that this is similar to other configs in handling hatchling so this is probably fine as well.